components/BottomNavigationBar.tsx

```diff
@@ -11,9 +11,14 @@ const BottomNavigationBar = () => {
   const pathname = usePathname();
 
   const isActive = (path: string) => {
-    return pathname === path;
+    if (path === '/') {
```

Comment:
Sorry, don't understand -- why was this needed?

Reply:
Just to make bottom navbar icons to be filled with color if they are active (even for nested routes).
So for content and class library, out route looks like: (/library/content?additionalParams). Therefore, we want the navbar icons remain activated (or filled with primary color) for these additional path segments.